CONTRIBUTING.md

```diff
@@ -111,7 +111,7 @@ The `--verbose` flag enables additional logs which will help to identify the sou
 
 ## Pull Request Guidelines
 
-Checkout a topic branch from a base branch, e.g. `main`, and merge back against that branch.
+Checkout a topic branch from a base branch, e.g. `master`, and merge back against that branch.
```

Comment:
We‘re currently using `main` as our primary branch, so this isn‘t accurate (we‘re in the process of dropping `master`; we still have it around temporarily while we close open PRs and such)